### PR TITLE
Fix: Refactor tests in `KeyStoreUtilTest` to avoid duplication (#472)

### PR DESCRIPTION
**Pull request by alex81sp**
Signed-off-by: Alejandro Martínez <alex1983bcn@gmail.com>

## Description

closes: #472 

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by alex81sp** [comment]
Questions:
- Is there is a header for 2020?
- Do you still use the author javadoc tag?

File: jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/KeyStoreUtilLoadPrivateKeyTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Stas Sukhanov
+ * @since 08.03.2017
+ */
```
Comment (rohanKanojia) [inline jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/KeyStoreUtilLoadPrivateKeyTest.java:29]: I think you are the author of this file :-) .

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
> Questions:
> 
> * Is there is a header for 2020?

Umm, I don't think we have any header for 2020. 

> * Do you still use the author javadoc tag?

We don't use author Javadoc tags. But I also don't remember if we had any kind of discussion to stop using these. I think it's based on personal preference. Right now author tags are present in legacy code which was inherited from fabric8 maven plugin.

File: jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/KeyStoreUtilLoadPrivateKeyTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    assertNotNull(privateKey);
+  }
+
+  private String getFile(String path) throws FileNotFoundException {
+    URL fileURL = KeyStoreUtilTest.class.getResource(path);
+    if (fileURL == null)
+      throw new FileNotFoundException("Required private key : '" + path + "' not found it test resource directory");
+    return fileURL.getFile();
+  }
+}
```
Comment (manusa) [inline jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/access/KeyStoreUtilLoadPrivateKeyTest.java:56]: nit: Can we make the copy-pasted method in `KeyStoreUtilTest` static (package private) and reuse it?

**Comment by manusa** [comment]
> We don't use author Javadoc tags. But I also don't remember if we had any kind of discussion to stop using these. I think it's based on personal preference. Right now author tags are present in legacy code which was inherited from fabric8 maven plugin.

I personally don't use them because I dislike the practice (VCS blame is usually more accurate defining authorship and doesn't get outdated). However, seeing how this project was fork-migrated (as specified in Eclipse handbook), there are still good reasons to use them. TL;DR Up to you.

**Review by manusa — APPROVED** [review]
LGTM! Thank you very much for your contribution

**Pull request merged**